### DISCUSSION TITLE
web端页面 在移动端打开会报错

### DISCUSSION THREAD

**Discussion by zxcv66666** [Tencent/libpag#1720] [Q&A]
![image](https://github.com/Tencent/libpag/assets/144442760/143cfda4-74e1-4ab4-b6c8-a4da1656089c)
以这种方式引入的wasm文件
在打开vconsole这个插件的时候，会有这种黄色的警告
<img width="419" alt="image" src="https://github.com/Tencent/libpag/assets/144442760/ba550fa1-f353-4e51-81a8-fd645b2a3f8b">
这个是因为我操作有问题，还是和vconsole插件有冲突呢
另外在ios以及浏览器上 都可以正常播放，但是在部分安卓手机上会爆出这种错误
<img width="409" alt="image" src="https://github.com/Tencent/libpag/assets/144442760/640988fc-40be-415c-abd9-283bddfca4c7">
但是我明明引入了wasm文件 ，在别的手机上也生效 想问一下为什么有的安卓会出现这种情况

**Comment by domchen** [comment]
与 #1724 提问重复

**Reply by yangchongduo** [reply]
怎么可能重复，ios   和web  没问题，安卓下有问题 ，肯定一套 代码
